src/cdk/schematics/ng-generate/drag-drop/index.ts

```diff
@@ -29,7 +34,11 @@ export default function (options: Schema): Rule {
 /** Adds the required modules to the main module of the CLI project. */
 function addDragDropModulesToModule(options: Schema) {
   return async (host: Tree) => {
-    const modulePath = await findModuleFromOptions(host, options);
-    addModuleImportToModule(host, modulePath!, 'DragDropModule', '@angular/cdk/drag-drop');
+    const isStandalone = await isStandaloneSchematic(host, options);
+
+    if (!isStandalone) {
```

Comment:
Having this check in each schematic is a little redundant, but IMO it's preferable to having `addModuleImportToModule` magically become a noop in a standalone app.